packages/plugins/i18n/admin/src/components/CMListViewInjectedComponents/DeleteModalAdditionalInfos/index.js

```diff
@@ -4,6 +4,14 @@ import { Typography } from '@strapi/design-system';
 import { getTrad } from '../../../utils';
 import useHasI18n from '../../../hooks/useHasI18n';
 
+const Emphasis = (chunks) => {
+  return (
+    <Typography fontWeight="semiBold" textColor="danger500">
+      {chunks}
+    </Typography>
+  );
+};
+
```

Comment:
Extracting this to its own component gets rid of an eslint warning about components being created during render